contrib/linux/actions/workflows/diag_loadavg.yaml

```diff
@@ -47,5 +43,5 @@
       name: "dump_results"
       ref: "core.local"
       parameters:
-        cmd: "echo \"ST2 Workflow\tdiag_loadavg:\t{{__results}}\" >> /tmp/diag_loadavg && echo 'Output written to file /tmp/diag_loadavg'"
+        cmd: "echo \"{{__results}}\" | python2 -c \"import json; import sys; from datetime import datetime; diag_data=eval(sys.stdin.read()); diag_data['timestamp'] = str(datetime.now()); print(json.dumps(diag_data))\" >> /tmp/diag_loadavg.json && echo 'Output written to file /tmp/diag_loadavg.json'"
```

Comment:
Looks like this is the only place with the hardcoded `python2` now.
Can we do something with it?

Reply:
Yeah, any reason why we need to use ``python2`` here? I believe that line should also work fine with Python 3 (aka just using ``python`` system binary - should work if it's either python 2 or python 3).

Reply:
I don't see any issues regarding py3 and it should work as well (I can put that on my list again). 

Let's also tackle the `eval`. The safe alternative without a bunch of JSON processing on stdin is ast.literal_eval: https://docs.python.org/3/library/ast.html#ast.literal_eval

Would this be fine for everyone? Last resort would be some manual parsing & processing with the JSON library.

Reply:
`ast.literal_eval` is awesome (and quite safe)! go for it.

Reply:
Is this sufficient?
```suggestion
        cmd: "echo \"{{__results}}\" | python3 -c \"import json; import sys; from datetime import datetime; diag_data=eval(sys.stdin.read()); diag_data['timestamp'] = str(datetime.now()); print(json.dumps(diag_data))\" >> /tmp/diag_loadavg.json && echo 'Output written to file /tmp/diag_loadavg.json'"
```